app/assets/stylesheets/responsive/_global_style.scss

```diff
@@ -237,3 +237,53 @@ div.pagination {
     color: #fff;
   }
 }
+
+
+/* Tooltips */
+
+.has-tooltip {
+  cursor: pointer;
+}
+
+.alaveteli-container {
+position: relative;
+}
+
+.alaveteli-tooltip {
+  background-color: #fafafa;
+  border: 1px solid #e9e9e9;
+  color: #333;
+  padding: 0.5em;
+  position: absolute;
+  z-index: 666;
+  top: -50%;
+  left: 100%;
+  max-width: 15em;
+  min-width: 10em;
+  p {
+    font-size: 0.736842105em; //14px
+    margin-bottom: 0;
+  }
+  a {
+    text-decoration: none;
+    &:hover,
+    &:active,
+    &:focus {
+      text-decoration: underline;
+    }
+  }
+}
+
+.alaveteli-tooltip__close {
+  display: none; //temporary until we get around to styling it
+}
+
+.simple-tooltip-tooltip__close {
+  float: right;
+  /** fix typo inputs **/
```

Comment:
I don't understand this comment – maybe some extra detail in the commit message would be good for explanation?